lib/rules/media-type-no-deprecated/__tests__/index.mjs

```diff
@@ -0,0 +1,126 @@
+import rule from '../index.mjs';
+
+const { messages, ruleName } = rule;
+
+testRule({
+	ruleName,
+	config: [true],
+	computeEditInfo: true,
+
+	accept: [
```

Comment:
minor

Can you add an accept test with no media type?
i.e. default to `all`

Reply:
Done